content/community/organization/policies.html

```diff
@@ -38,13 +38,28 @@ <h4>Coding Guidelines</h4>
 <HR>
 <h2>Communication</h2>
```

Comment:
I think it makes sense to put this at the top of the page, rather than the bottom.